cpp/test/util_test/from_file_test.cc

```diff
@@ -112,7 +112,7 @@ TYPED_TEST(FromFileTest, IfFileExistsRowsAndColsMethod) {
 }
 
 TYPED_TEST(FromFileTest, CSVRowsAndColsMethodTest) {
-  this->Filer("../test/data_for_test/matrix_2_2_csv.txt", 2, 2, ',');
+  this->Filer("../test/data_for_test/matrix_2_2_csv.txt", 2, 2, ",");
```

Comment:
Could you make a directory named FromFile under ../test/data_for_test/ and put the corresponding test data files in there?